### PR TITLE
Chrissi Solutions

### PR DESCRIPTION
**Pull request by chmette**
Please review my excercise

### CODE REVIEW

File: chrissi/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+let snackServings = 4;
+let guests = 16;
+
+console.log(snackServings, guests);
+
+// 1 ADD
+console.log(snackServings + guests);
+
+// 2 SUB
+console.log(snackServings - guests);
```
Comment (ciamac-da) [inline chrissi/index.js:11]: You forgot to subtract snackServings from guests!
all you need is to type  console.log(guests - snackServings);

File: chrissi/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// 4 DIV
+console.log(snackServings / guests);
+console.log(guests / snackServings);
+
+// 5
+let drinks = 10;
+
+// (snackServings * guests) = 64
+
+console.log(64 / drinks);
```
Comment (ciamac-da) [inline chrissi/index.js]: You have to get rid of 64 and also console.log(64 / drinks). Just make a variable  snackServings * guests instead and show the result in console and the code should be like this =>
const resultOne = (snackServings * guests) / drinks;
console.log(resultOne)

File: chrissi/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+let monkeyfights = 20;
+
+console.log((monkeys + bananas) * monkeyfights);
+
+let resultTwo = 480;
+console.log(resultTwo);
+
+// 8
+
```
Comment (ciamac-da) [inline chrissi/index.js:39]: 8th and 9th tasks answers are wrong! And the solution is :
8th- console.log(++monkeys)
9th- console.log(--bananas)

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Thanks a lot for pushing you Code Christin! Please read my comments, that i wrote and try to fix the code and push again:)

File: chrissi/index.js (hunk 1 above)
Comment (chmette) [reply chrissi/index.js:11]: done, i overread it

File: chrissi/index.js (hunk 2 above)
Comment (chmette) [reply chrissi/index.js]: done

File: chrissi/index.js (hunk 3 above)
Comment (chmette) [reply chrissi/index.js:39]: we did NOT have this in the lesson

**Comment by chmette** [comment]
@ciamac-da please see my answers to your comments

**Review by ciamac-da — APPROVED** [review]
Cool, thanks:)

File: chrissi/index.js (hunk 3 above)
Comment (ciamac-da) [reply chrissi/index.js:39]: Yeah I know, sorry for that!

**Pull request merged**